junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/DefaultWebTestClient.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URI;
+
+import com.linecorp.armeria.client.BlockingWebClient;
+import com.linecorp.armeria.client.ClientOptions;
+import com.linecorp.armeria.client.HttpClient;
+import com.linecorp.armeria.client.RequestOptions;
+import com.linecorp.armeria.client.endpoint.EndpointGroup;
+import com.linecorp.armeria.common.ExchangeType;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.Scheme;
+
+
+final class DefaultWebTestClient implements WebTestClient {
+
+    static final WebTestClient DEFAULT = new DefaultWebTestClient(BlockingWebClient.of());
+
+    static final RequestOptions RESPONSE_STREAMING_REQUEST_OPTIONS =
```

Comment:
Is this used?

Reply:
I created this variable to use the execute(HttpRequest req) method in the TestHttpResponse class, which is a reference to the execute method in the WebClient interface. It could be possible not to use it because execute method of `BlockingWebClient` does not use it. I will remove this static variable and modify the execute method.